boot.go

```diff
@@ -11,11 +11,15 @@ import (
 	"github.com/deis/builder/pkg/cleaner"
 	"github.com/deis/builder/pkg/conf"
 	"github.com/deis/builder/pkg/gitreceive"
-	"github.com/deis/builder/pkg/gitreceive/storage"
 	"github.com/deis/builder/pkg/healthsrv"
 	"github.com/deis/builder/pkg/sshd"
 	"github.com/deis/builder/pkg/sys"
 	pkglog "github.com/deis/pkg/log"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	_ "github.com/docker/distribution/registry/storage/driver/azure"
+	"github.com/docker/distribution/registry/storage/driver/factory"
+	_ "github.com/docker/distribution/registry/storage/driver/gcs"
+	_ "github.com/docker/distribution/registry/storage/driver/s3-aws"
```

Comment:
+1 to @krancour's statement. We should at least use [our fork](https://github.com/deis/distribution), but I'd prefer a new repository that is 100% dedicated to this storage abstraction. Note that @mboersma has submitted https://github.com/docker/distribution/issues/1537, which is highly related.

Finally, I'd prefer that, eventually, we refactor whatever storage library that we work with to:

- Not use anonymous imports unless for good, documented reason (I don't see any documentation anywhere at or under https://godoc.org/github.com/docker/distribution/registry/storage/driver)
- Not use the factory pattern (I don't see a use for it based on https://godoc.org/github.com/docker/distribution/registry/storage/driver/factory)